mobify/platform-scaffold
# [IM to SDK] Move product store branch to SDK

This PR moves the products branch of the Redux store into the SDK. 

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-21
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/698

## Changes
- Remove products branch of the store
- Update paths to reference products branch in the SDK

## How to test-drive this PR
- checkout the move-products-tore-to-sdk branch for the SDK
- cd into the SDK folder
- run `npm link`
- run `npm run dev:build`
- cd back to the scaffold
- run `npm link progressive-web-sdk`
- run `npm run dev`
- preview the merlins site
- navigate to a PLP and PDP
- confirm everything still works as expected
